packages/ember-glimmer/lib/environment.js

```diff
@@ -68,7 +88,8 @@ export default class Environment extends GlimmerEnvironment {
         let definition = this.getComponentDefinition(path);
 
         if (definition) {
-          return new CurlyComponentSyntax({ args, definition, templates });
+          wrapClassAttribute(args);
```

Comment:
is there a way to achieve this in `args` setters/getters for `'class'`? It would be more scalable and, potentially, slightly more performant (won't call wrapper against every component)